lib/vagrant-libvirt/plugin.rb

```diff
@@ -23,7 +23,7 @@ class Plugin < Vagrant.plugin('2')
         Config
       end
 
-      provider('libvirt', parallel: true) do
+      provider('libvirt', parallel: true, box_optional: true) do
```

Comment:
I don't believe this will have any effect for vagrant 1.5, won't cause any compatibility issues AFAIK, will just be ignored. Maybe just mention in the docs that this requires vagrant 1.6+ to be usable?